docs/machine-learning/tutorials/image-classification.md

```diff
@@ -111,336 +111,249 @@ toaster2.png	appliance
+    The following steps in the tutorial add the steps to construct, train, and evaluate the model.
 
-`softmax2_pre_activation` assists the model with determining which class the images belongs to. `softmax2_pre_activation` returns a probability for each of the categories for an image, and all of those probabilities must add up to 1. It assumes that an image will belong to only one category, as shown in the following example:
+## Construct the ML.NET model pipeline
 
-| Class         | Probability   |
-| ------------- | ------------- |
-| `Food`        |  0.001        |
-| `Toy`         |  0.95         |
-| `Appliance`   |  0.06         |
+An ML.NET model pipeline is a chain of estimators. Note that no execution happens during pipeline construction. The estimator objects are created but not executed.
 
-Append the `TensorFlowTransform` to the `estimator` with the following line of code:
+1. Add the estimators to load, resize and extract the pixels from the image data:
 
-[!code-csharp[ScoreTensorFlowModel](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#ScoreTensorFlowModel)]
+    [!code-csharp[ImageTransforms](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#ImageTransforms)]
 
-### Choose a training algorithm
+    The image data needs to be processed into the format that the TensorFlow model expects. In this case, the images are loaded into memory, resized to a consistent size, and the pixels are extracted into a numeric vector.
 
-To add the training algorithm, call the `mlContext.MulticlassClassification.Trainers.LbfgsMaximumEntropy()` wrapper method.  The [LbfgsMaximumEntropy](xref:Microsoft.ML.Trainers.LbfgsMaximumEntropyMulticlassTrainer) is appended to the `estimator` and accepts the Inception image features (`softmax2_pre_activation`) and the `Label` input parameters to learn from the historic data.  Add the trainer with the following code:
+1. Add the estimator to load the TensorFlow model, and score it:
 
-[!code-csharp[AddTrainer](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#AddTrainer)]
+    [!code-csharp[ScoreTensorFlowModel](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#ScoreTensorFlowModel)]
 
-You also need to map the `predictedlabel` to the `predictedlabelvalue`:
+    The next stage in the pipeline is to load the TensorFlow model into memory, then process the vector of pixel values through the TensorFlow model network. Applying inputs to a deep learning model is referred to as **Scoring**. When using the model in its entirety, scoring makes an inference, or prediction. 
```

Comment:
I would reword the last two sentences to clarify scoring as the process of getting a prediction, instead of applying inputs.

Reply:
Fixed.